### PR TITLE
A tool for automatically set marker-color

### PR DESCRIPTION
**Pull request by xavierchow**
It's for #2 , see: https://github.com/xavierchow/awesome-cn-cafe/tree/master/tool

### CODE REVIEW

**Comment by xavierchow** [comment]
Also add cafe icon for marker, sth like below, if you guys like it :smile: 
screenshot 2016-02-18 00 10 03

File: beijing.geojson [hunk 1]
```diff
@@ -8,7 +8,10 @@
         "下载速度": "22 Mbps",
         "Speedtest 链接": "http://www.speedtest.net/my-result/4629928302",
         "参考价格": "30-40 元",
-        "评论 (@xhacker)": "有电源，网速快，环境不错，咖啡也不错。"
+        "评论 (@xhacker)": "有电源，网速快，环境不错，咖啡也不错。",
+        "marker-color": "#50C240",
+        "marker-symbole": "cafe",
```
Comment (xhacker) [inline beijing.geojson]: Is `marker-symbole` a mistake?

File: beijing.geojson [hunk 2]
```diff
@@ -53,4 +62,4 @@
       }
     }
   ]
-}
+}
```
Comment (xhacker) [inline beijing.geojson]: Can we generate a new line here?

File: tool/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+## Prequisite
+[Node.js](https://nodejs.org/)
+
+## How to run
+
+`node mark_color.js <city>`
+
+e.g.
+
+`node mark_color.js shanghai`
```
Comment (xhacker) [inline tool/README.md]: Can we iterate through every `.geojson` file instead of requiring a parameter?

**Comment by xhacker** [comment]
Awesome job! Thank you for your contribution. Just a few minor comments 😆

File: beijing.geojson (hunk 1 above)
Comment (xavierchow) [inline beijing.geojson]: Sorry, it's a typo, this line is useless, will fix.

File: beijing.geojson (hunk 2 above)
Comment (xavierchow) [inline beijing.geojson]: Sure, it was auto generated json, I forgot to add EOL :blush:

File: tool/README.md (hunk 3 above)
Comment (xavierchow) [inline tool/README.md]: No problem.

**Comment by xavierchow** [comment]
Updated, cheers!

**Comment by xhacker** [comment]
Awesome, thank you!

**Pull request merged**